docs/ossm/README.md

```diff
@@ -0,0 +1,9 @@
+[Return to Sail Operator documentation](../)
+
+# User Documentation
+
+This documentation is specific to the OpenShift Service Mesh product and may differ slightly from the [Sail Operator documentation](../README.md). For OpenShift Service Mesh 3, this documentation should be prioritized over the [Sail Operator documentation](../README.md). However, the official source of truth is the product documentation available at [docs.redhat.com](https://docs.redhat.com). If a particular topic is not covered here, you can refer to the [Sail Operator documentation](../README.md) or [Istio.io](https://istio.io) documentation as supplementary resources.
```

Comment:
```suggestion
This documentation is specific to the OpenShift Service Mesh product and may differ slightly from the [Sail Operator documentation](../README.md). For OpenShift Service Mesh 3, this documentation should be prioritized over the [Sail Operator documentation](../README.md). However, the official source of truth is the product documentation available at [docs.redhat.com](https://docs.openshift.com). If a particular topic is not covered here, you can refer to the [Sail Operator documentation](../README.md) or [Istio.io](https://istio.io) documentation as supplementary resources.
```

Reply:
Just noticed the comment from Gwynne regarding the decomissioning of the url. Approving the PR. Thanks Nick.